.gitignore

```diff
@@ -42,26 +38,37 @@ packages/*/*/*.d.ts
 packages/**/*.test-vrt.ts
 *.tsbuildinfo
 
+# ts + css assets
+src/**/*.css.ts
```

Comment:
This is repeated in line 24.

Reply:
is `src/**/*.css.ts` the same as `packages/*/src/**/*.css.js`?

Reply:
Either it _is_ the same or it points to something that no longer exists... I think there was an `src` directory in the root at some point 🤔. If the shorter version covers, it may be nice that is applies to both `packages` AND `projects` but that auto coverage might be unexpected. Thoughts?

Reply:
I don't believe it will, it would need the `*/src` for it to delve into parent folders.  This would exclusively apply to a src directory in the root.

Reply:
Then remove we shall, as in that case it references no files!